drivers/SmartThings/zwave-switch/src/zooz-zen-30-dimmer-relay/init.lua

```diff
@@ -11,12 +11,13 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
-
+local st_device = require "st.device"
 local capabilities = require "st.capabilities"
 local cc = require "st.zwave.CommandClass"
 local CentralScene = (require "st.zwave.CommandClass.CentralScene")({ version = 1 })
-local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 2 })
-local SwitchMultilevel = (require "st.zwave.CommandClass.SwitchMultilevel")({ version = 4 })
+local Basic = (require "st.zwave.CommandClass.Basic")({ version = 1, strict = true })
+local SwitchBinary = (require "st.zwave.CommandClass.SwitchBinary")({ version = 2, strict = true })
+local SwitchMultilevel = (require "st.zwave.CommandClass.SwitchMultilevel")({ version = 4, strict = true })
```

Comment:
can I ask why you're changing all of these to strict?

Reply:
To use the same version of cc as defaults. If I would not use strict it may end up with some stupid issues in unit tests like not known arg name value/target_value etc.

Reply:
I think you're running into this: https://github.ecodesamsung.com/iot-hub/scripting-engine/commit/12a8e210bc2a9aef6c01b6ae85d5484b83de14b8

The defaults now prefer `value` rather than `target_value` because that's what the DTHs did. I have a PR open that changes all the tests that used `target_value` for when 45 is available publicly.